src/Seksjon/Utdanning.elm

```diff
@@ -0,0 +1,193 @@
+    | Oppsummering NivåRetningSkoleBeskrivelsePeriodeOppsummeringInfo
+
+
+type alias NivåInfo =
+    { nivå : String }
+
+
+type alias NivåRetningInfo =
+    { nivå : String, retning : String }
+
+
+type alias NivåRetningSkoleInfo =
+    { nivå : String, retning : String, skole : String }
+
+
+type alias NivåRetningSkoleBeskrivelseInfo =
+    { nivå : String, retning : String, skole : String, beskrivelse : String }
+
+
+type alias NivåRetningSkoleBeskrivelsePeriodeInfo =
+    { nivå : String, retning : String, skole : String, beskrivelse : String, periode : String }
+
+
+type alias NivåRetningSkoleBeskrivelsePeriodeOppsummeringInfo =
+    { nivå : String, retning : String, skole : String, beskrivelse : String, periode : String, oppsummering : String }
+
+
+type Msg
+    = BrukerVilRegistrereNyUtdanning
+    | GåTilArbeidserfaring
+    | BekreftAlleredeRegistrert
+    | BrukerVilRegistrerNivå String
+
+
+type SamtaleStatus
+    = IkkeFerdig ( Model, Cmd Msg )
+    | Ferdig (List Utdanning) MeldingsLogg
+
+
+meldingsLogg : Model -> MeldingsLogg
+meldingsLogg (Model model) =
+    model.seksjonsMeldingsLogg
+
+
+
+--- UPDATE ---
+
+
+update : Msg -> Model -> SamtaleStatus
+update msg (Model model) =
+    case msg of
+        BrukerVilRegistrereNyUtdanning ->
+            case model.aktivSamtale of
+                FullførtRegistrering ->
+                    IkkeFerdig ( Model model, Cmd.none )
+
+                _ ->
+                    IkkeFerdig ( Model model, Cmd.none )
+
+        {--
+            ( model.aktivSamtale
+                |> RegistrerNivå tekst
+                |> nesteSamtaleSteg model "Endre"
+            , Cmd.none
+            )
+                |> IkkeFerdig
+--}
+        GåTilArbeidserfaring ->
+            Ferdig model.utdanningListe model.seksjonsMeldingsLogg
+
+        _ ->
+            IkkeFerdig ( Model model, Cmd.none )
+
+
+
+{--Ferdig model.utdanning model.meldingsLogg--}
+
+
+nesteSamtaleSteg : ModelInfo -> String -> Samtale -> Model
+nesteSamtaleSteg model tekst samtaleSeksjon =
+    Model
+        { model
+            | aktivSamtale = samtaleSeksjon
+        }
+
+
+oppdaterSamtaleSteg : ModelInfo -> Samtale -> Model
+oppdaterSamtaleSteg model samtaleSeksjon =
+    Model
+        { model
+            | aktivSamtale = samtaleSeksjon
+        }
+
+
+samtaleTilMeldingsLogg : Samtale -> List Melding
+samtaleTilMeldingsLogg utdanningSeksjon =
+    case utdanningSeksjon of
+        Intro ->
+            [ Melding.spørsmål
+                [ "Intro" ]
+            ]
+
+        FullførtRegistrering ->
+            [ Melding.spørsmål
+                [ "Du har valgt å endre personaliaskjema" ]
+            ]
+
+        RegistrerUtdanning ->
+            [ Melding.spørsmål
+                [ "Du har valgt å endre personaliaskjema" ]
+            ]
+
+        _ ->
+            [ Melding.spørsmål
+                [ "Diverse valg" ]
+            ]
+
+
+viewUtdanning : Model -> Html Msg
+viewUtdanning (Model { aktivSamtale }) =
```

Comment:
Denne kan godt hete `viewBrukerInput` sånn som den gjør i personaliaseksjonen. Og her burde det bare være knapper og inputfelter og sånt for det meste. Ikke tekst.